public/docs/_examples/pipes/dart/lib/fetch_json_pipe.dart

```diff
@@ -1,18 +1,22 @@
+// #docregion
 import 'dart:html';
 import 'dart:async';
 import 'dart:convert';
 
 import 'package:angular2/angular2.dart';
 
+// #docregion pipe-metadata
 @Pipe(name: 'fetch', pure: false)
-@Injectable()
+// #enddocregion pipe-metadata
+@Injectable() // FIXME(chalin): unnecessary?
 class FetchJsonPipe extends PipeTransform {
   dynamic _fetchedValue;
   Future<dynamic> _fetchPromise;
-  transform(dynamic value, [List<dynamic> args]) {
+
+  transform(String url) {
```

Comment:
Yes, that bothered me too, but given that this example doesn't run anymore in beta.16, I decided to stop making changes. The entire example needs revision anyhow. For now, I need to focus on getting the Dart version of the prose out. I will revise this example once I get to the pipes chapter.